go/api/handlers/note.go

```diff
@@ -67,6 +67,10 @@ func (handler *NoteHandler) GetNotesOfTrip(context *gin.Context) {
 			ID:      note.ID,
 			Title:   note.Title,
 			Content: note.Content,
+			Author: responses.UserRoleReponse{
+				Username: note.Author.Username,
+				Role:     note.Author.Role,
+			},
```

Comment:
Utilise plutot le UserReponse et passe l'id

https://github.com/ESGI-69/challenge-flutter-go/blob/a4f7fa04c01c062017cc2db07f9931f28337b7f6/go/api/responses/user.go#L5-L8

```suggestion
			Author: responses.UserReponse{
			  ID: note.Author.ID
				Username: note.Author.Username,
			},
```